src/create-card.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+const styles = {
+  background: {
+    backgroundColor: '#fffbe0',
+    width: '32rem'
+  }
+}
+
+export default function CreateCard() {
+  return (
+    <div className="d-flex justify-content-center">
+      <form id="createFlash" className="border border-dark rounded mt-5 p-4" style={styles.background}>
```

Comment:
Bold/dark borders are really risky. The default looks good enough. If you really want a custom theme. Choose one from here: https://bootswatch.com/
They're on a CDN too.
https://cdnjs.com/libraries/bootswatch